migrations/2022_05_14_181416_create_comment_user_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('comment_user', function (Blueprint $table) {
```

Comment:
One last think I now understood is that you are using this m2m table to have the relation for likes? the name is not ideal although aligns with laravel standards. we need though to take control of this and have a name that is better fit. Maybe something like `comment_user_likes`

Reply:
ok renamed it as comment_user_likes